device/mellanox/x86_64-mlnx_msn4600-r0/ACS-MSN4600/port_config.ini

```diff
@@ -0,0 +1,65 @@
+# name          lanes               alias     index
```

Comment:
Are you sure this is aligned with hw? you need 4 lanes for 200G, Please check how it was done in Onyx.

Reply:
I check with Onyx and with HW and they confirmed that Liger uses 4 lanes for 200G